addon/modifiers/style.js

```diff
@@ -1,4 +1,4 @@
-import Ember from 'ember';
+import { setModifierManager, capabilities } from '@ember/modifier';
```

Comment:
Is `ember-modifier-manager-polyfill` supporting that API? In it's documentation it's only using `Ember._setModifierManager()`. If not I would be also fine with dropping support for Ember < 3.13.

Reply:
It is supposed to work with older Ember as `ember-modifier-manager-polyfill` polyfills those APIs. I think the documentation is out of date. I took as a reference https://github.com/emberjs/ember-render-modifiers which is also using the polyfill.

Reply:
In that case we might want to add Ember 2.18 back to the supported versions? Only if it does not cause any additional work. This one would be a breaking change anyways cause it drops support for node 6.

Reply:
Returned 2.18 to the ember-try and travis configs